backend/api/visualization.py

```diff
@@ -0,0 +1,53 @@
+import pandas as pd
+
+
+def generate_chords(modality: str, cohorts: list[str]):
```

Comment:
Would be good to have tests for this function, optionally great would be tests for the api endpoints as well, @cebel has some experience with that and might able to give you some pointers

Reply:
I added a test and a testing workflow for Python 3.10 and above